### PR TITLE
Modifying hip library version and error code types

### PR DESCRIPTION
**Pull request by Nexengineer**

### CODE REVIEW

File: src/In.ProjectEKA.OtpService/Otp/OtpController.cs [hunk 1]
```diff
@@ -37,7 +38,7 @@ private ActionResult ReturnServerResponse(OtpResponse otpResponse)
                 ResponseType.OtpInvalid => BadRequest(otpResponse),
                 ResponseType.OtpExpired => Unauthorized(otpResponse),
                 ResponseType.OtpGenerationFailed => BadRequest(otpResponse),
```
Comment (ganesanarun) [inline src/In.ProjectEKA.OtpService/Otp/OtpController.cs]: If the generation failed, isn't it a server problem?

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ test/In.ProjectEKA.HipServiceTest/Link/LinkPatientControllerTest.cs  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4913134&bid=16263246)

File: src/In.ProjectEKA.OtpService/Otp/OtpController.cs (hunk 1 above)
Comment (ganesanarun) [reply src/In.ProjectEKA.OtpService/Otp/OtpController.cs]: Should the OTP API be referenced as /otp/link or can't that just be /otp?
Should the OTP VERIFY API be referenced as /otp/verify or can't that just be /otp/{sessionID}/verify?

Comment (Nexengineer) [reply src/In.ProjectEKA.OtpService/Otp/OtpController.cs]: Should the OTP API be referenced as /otp/link or can't that just be /otp: Changed

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ test/In.ProjectEKA.HipServiceTest/Link/LinkPatientControllerTest.cs  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4913134&bid=16263246)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ test/In.ProjectEKA.HipServiceTest/Link/LinkPatientControllerTest.cs  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4913134&bid=16263246)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ test/In.ProjectEKA.HipServiceTest/Link/LinkPatientControllerTest.cs  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4913134&bid=16263246)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ test/In.ProjectEKA.HipServiceTest/Link/LinkPatientControllerTest.cs  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4913134&bid=16263246)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ test/In.ProjectEKA.HipServiceTest/Link/LinkPatientControllerTest.cs  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4913134&bid=16263246)

**Pull request merged**